plugins/serializers/registry.go

```diff
@@ -2,14 +2,16 @@ package serializers
 
 import (
 	"fmt"
-	"github.com/influxdata/telegraf/plugins/serializers/prometheusremotewrite"
 	"time"
 
+	"github.com/influxdata/telegraf/plugins/serializers/prometheusremotewrite"
```

Comment:
If you move this, then please insert in the list below in alphabetical order.

Reply:
Ack.